cobra/test/test_io/test_annotation_format.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from os.path import join
+from cobra.io import load_json_model, write_sbml_model
+
+
+def test_load_json_model_valid(data_directory):
+    """Testing valid annotation format"""
+    path_to_file = join(data_directory, "valid_annotation_format.json")
+    model = load_json_model(path_to_file)
+    dict = {
+        'bigg.reaction': [['is', 'PFK26']],
+        'kegg.reaction': [['is', 'R02732']],
+        'rhea': [['is', '15656']]
+    }
+    for x in model.metabolites:
+        assert x.annotation == dict, "Dictionary is not equal"
```

Comment:
The `pytest` suite already provides deep inspection of the assertion so the message is not necessary.

```suggestion
        assert metabolite.annotation == expected
```